prow/bugzilla/client.go

```diff
@@ -201,7 +201,7 @@ func getRootForClone(c Client, bug *Bug) (*Bug, error) {
 		}
 		switch l := len(parent); {
 		case l <= 0:
-			break
+			return curr, utilerrors.NewAggregate(errs)
```

Comment:
testcase?
/ok-to-test

Reply:
I didn't need to add a testcase to tackle it, just needed to add a dependent bug which I have added